my-site/orange.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles/global.css" />
+  </head>
+  <body>
+    This color was added by
+    <ul>
+      <li>Orange Team</li>
```

Comment:
Please replace `Orange Team` by its member names.

For more references, please read the milestone description here: https://github.com/aeatencio/pick-a-color/milestone/1